safe/utilities/test/test_rounding.py

```diff
@@ -78,5 +77,44 @@ def test_fatalities_range(self):
         self.assertEqual('10,000 - 100,000', fatalities_range(18000))
         self.assertEqual('> 100,000', fatalities_range(101000))
 
+    def test_denomination(self):
+        """Test name number."""
+        result = denomination(1)
+        self.assertEqual(1, result[0])
+        self.assertEqual('unit_ones', result[1]['key'])
+
+        result = denomination(11)
+        self.assertEqual(1.1, result[0])
+        self.assertEqual('unit_tens', result[1]['key'])
+
+        result = denomination(99)
+        self.assertEqual(9.9, result[0])
+        self.assertEqual('unit_tens', result[1]['key'])
+
+        result = denomination(100)
+        self.assertEqual(1, result[0])
+        self.assertEqual('unit_hundreds', result[1]['key'])
+
+        result = denomination(101)
+        self.assertEqual(1.01, result[0])
+        self.assertEqual('unit_hundreds', result[1]['key'])
+
+        result = denomination(1001)
+        self.assertEqual(1.001, result[0])
+        self.assertEqual('unit_thousand', result[1]['key'])
+
+        result = denomination(10000)
+        self.assertEqual(10, result[0])
+        self.assertEqual('unit_thousand', result[1]['key'])
+
+        result = denomination(101000)
+        self.assertEqual(101, result[0])
+        self.assertEqual('unit_thousand', result[1]['key'])
+
+        result = denomination(100000000000000)
```

Comment:
Can you put test for non positive integer? Like float, or negative number? Just in case we have a bug, then the number is not positive integer, then we are still sure that it won't throw python exception.